features/support/hooks.rb

```diff
@@ -1,9 +1,18 @@
 Before('@sudo') do
   fail 'sudo authentication failed' unless system 'sudo -v'
   @aruba_timeout_seconds = 15
+  ENV['TREMA_LOG_DIR'] = '.'
+  ENV['TREMA_PID_DIR'] = '.'
+  ENV['TREMA_SOCKET_DIR'] = '.'
 end
 
 After('@sudo') do
-  run 'trema killall'
-  sleep 10
+  begin
+    run 'trema killall --all'
+    sleep 10
+  ensure
+    ENV['TREMA_LOG_DIR'] = nil
+    ENV['TREMA_PID_DIR'] = nil
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.